bin/bails-menu

```diff
@@ -33,24 +33,26 @@ export DATA_DIR="/live/persistence/TailsData_unlocked/Persistent/.bitcoin"
 export BAILS_DIR="$LOCAL_DIR"
 
 onlynet_onion() {
-zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
+    zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.
 
-Would you like to enhance privacy by connecting exclusively to Tor (onion) peers with reduced clearnet exposure? This may limit your network diversity and can be less stable if you have limited Tor peers.' --icon-name=tor-connection --default-cancel
+    Would you like to enhance privacy by connecting exclusively to Tor (onion) peers with reduced clearnet exposure? This may limit your network diversity and can be less stable if you have limited Tor peers.' --icon-name=tor-connection --default-cancel
 
-if [ $? = 0 ]; then
-    # User chose to enable -onlynet=onion
-    sed --in-place 's/#onlynet=<net>/onlynet=onion/' $DATA_DIR/bitcoin.conf
-    zenity --info --title='Network settings updated' --text='You have chosen to connect exclusively to Tor (onion) peers.
+    zenity_exit_status=$?
```

Comment:
why create a new variable here, $? is the exit status.

Reply:
I added a note as the description to that commit (eefb03b9c2bf621d5cdc8d5362411bbc909be4d1). It was required to satisfy shellcheck. We might want to revert this commit and exclude SC2181.
https://www.shellcheck.net/wiki/SC2181

The other option is something like this, but it's harder to find where the `; then` part is.
```
if zenity --question --title='Network Privacy' --width=420 --text='Bails offers different settings for connecting to the Bitcoin network. By default, Bails connects to both clearnet and Tor (onion) peers, providing a balance between usability and privacy.

Would you like to enhance privacy by connecting exclusively to Tor (onion) peers with reduced clearnet exposure? This may limit your network diversity and can be less stable if you have limited Tor peers.' --icon-name=tor-connection --default-cancel; then
    # User chose to enable -onlynet=onion
    sed --in-place 's/#onlynet=<net>/onlynet=onion/' $DATA_DIR/bitcoin.conf
    zenity --info --title='Network settings updated' --text='You have chosen to connect exclusively to Tor (onion) peers.

Your network settings have been updated.' --width=420 --icon-name=tor-connected-symbolic
...
```

Reply:
could put `then` on a new line.

Or smarter, put the text in a variable (defined right before it is used for clairty) so the command fits on one line. Tails does this all the time in their Shell.

That check wasn't

Reply:
I agree creating a variable for the multi line string will be a clean solution. Will do.